base/docs/Docs.jl

```diff
@@ -201,7 +201,7 @@ end
 
 doc(f, ::Method) = doc(f)
 
-# Modules
+# Modules
```

Comment:
This is pretty weird, have we somehow ended up with a different kind of space here?